app/clients/edamam_client.rb

```diff
@@ -0,0 +1,10 @@
+class EdamamClient
+  include HTTParty
+  base_uri 'https://api.edamam.com/search'
+  default_params app_id: ENV['EDAMAM_APP_ID'], app_key: ENV['EDAMAM_APP_KEY']
```

Comment:
It'd be nice to have consistent `ENV` variable naming. Currently, we have the following schemes:

- `<NAME>_API_KEY`
- `<NAME>_KEY`, `<NAME>_SECRET`
- `<NAME>_APP_ID`, `<NAME>_APP_KEY`

I propose we stick to something like `<NAME>_API_<WHATEVER>`, i.e.

```
# in .env
MOVIEDB_API_KEY=...
DISCOGS_API_KEY=...
DISCOGS_API_SECRET=...
EDAMAN_API_ID=...
EDAMAN_API_KEY...
...
```